src/main/java/frc/robot/bindings/KeyboardBindings.java

```diff
@@ -0,0 +1,15 @@
+package frc.robot.bindings;
+
+import frc.robot.Robot;
+import frc.utils.controllers.keyboard.KeyboardController;
+
+public class KeyboardBindings {
+
+	private static final KeyboardController KEYBOARD_CONTROLLER = new KeyboardController();
+
+	public static void configureBindings(Robot robot) {
+		KeyboardController usedKeyboard = KEYBOARD_CONTROLLER;
+		// bindings
```

Comment:
```suggestion
		// bindings...
```

Reply:
Should I change also for the joysticks in `frc.robot.bindings.JoySticksBindings`?